### DISCUSSION TITLE
Unable to set kitty as my default terminal with update-alternatives

### DISCUSSION THREAD

**Discussion by eXsoR65** [kovidgoyal/kitty#3727] [Q&A] [answered]
I installed Kitty from the Binary and integrated it with gnome but when trying to use `update-alternatives --config x-terminal-emulator` it not listed as an option.

Pop_OS 20.10 (Ubuntu based) with gnome. 

Any help with this is appreciated.

**Comment by kovidgoyal** [comment]
Nto a ubuntu/debian user so not familiar with update-alternatives, but most likley you just need to get the kitty.desktop file you created into the relevant databse. Maybe by calling xdg-mime or similar.

**Reply by eXsoR65** [reply]
yeah looking in to it more, update-alternatives is for all Debian based distros and it basically crates a LinkName to the bin for the whatever software you want as default. I was hoping to get help from someone that has done it before as I have not been able to get it to work.

**Comment by jmcdo29** [comment]
Sorry to bring up an old thread, I was able to solve this by using the `update-alternatives --install` command like so:

```
sudo update-alternatives --install /usr/bin/x-terminal-emulator x-terminal-emulator <path to kitty bin> 10
```

After the install I was able to run `sudo update-alternatives --config x-terminal-emulator` and choose `kitty` as my terminal emulator of choice

**Reply by stefumies** [reply, marked as answer]
Seems this command is not correct, I am getting:
`update-alternatives: --install needs <link> <name> <path> <priority>`

**Reply by Nanito111** [reply]
Thanks! This worked for me
(Fedora 39 | Gnome 45.5)
